cdap-common/src/main/java/io/cdap/cdap/common/ssh/SSHConfig.java

```diff
@@ -28,15 +30,17 @@
 
   private final String host;
   private final int port;
+  private final InetSocketAddress proxyAddress;
 
   private final String user;
   private final Supplier<byte[]> privateKeySupplier;
   private final Map<String, String> configs;
 
-  private SSHConfig(String host, int port, String user,
+  private SSHConfig(String host, int port, InetSocketAddress proxyAddress, String user,
```

Comment:
Should it be marked `@Nullable` or not required because its a private constructor?

Reply:
Updated